src/detect-http-client-body.c

```diff
@@ -76,6 +76,219 @@ static uint8_t DetectEngineInspectBufferHttpBody(DetectEngineCtx *de_ctx,
 static int PrefilterMpmHttpRequestBodyRegister(DetectEngineCtx *de_ctx, SigGroupHead *sgh,
         MpmCtx *mpm_ctx, const DetectBufferMpmRegistery *mpm_reg, int list_id);
 
+static inline InspectionBuffer *FiledataWithXformsGetDataCallback(DetectEngineThreadCtx *det_ctx,
```

Comment:
this looks like a copy of file data code. Can we just reuse that?

Reply:
Yes it is.
Is there something even simpler to do ? Like for detect-http-server-body.c which uses g_file_data_buffer_id directly ?

Reply:
Maybe, it would require a bit of investigation.

Reply:
Ok, so going for simple reuse of code, not bigger changes in v3...